controllers/mdutil/util.go

```diff
@@ -414,6 +416,7 @@ func EqualMachineTemplate(template1, template2 *clusterv1.MachineTemplateSpec) b
 }
 
 // FindNewMachineSet returns the new MS this given deployment targets (the one with the same machine template).
+// Deprecated: this does not for rolloutAfter. Use FindNewMachineSetWithRolloutAfter.
 func FindNewMachineSet(deployment *clusterv1.MachineDeployment, msList []*clusterv1.MachineSet) *clusterv1.MachineSet {
```

Comment:
If we wait for v1.0, we can make this a breaking change and change the signature of this and the other function.

Both functions take the exact same inputs, we could have an Input struct instead?

Reply:
Also I actually question they should be public.